lib/msal-core/src/error/ClientAuthError.ts

```diff
@@ -88,6 +88,10 @@ export const ClientAuthErrorMessage = {
     invalidInteractionType: {
         code: "invalid_interaction_type",
         desc: "The interaction type passed to the handler was incorrect or unknown"
+    },
+    cacheParseError: {
+        code: "cannot_parse_cache",
+        desc: "The cached token key is not a JSON, cannot be parsed"
```

Comment:
Typo, "The cached token key is not valid JSON and connect be parsed"

Reply:
Done.